internal/access/credential.go

```diff
@@ -86,10 +86,16 @@ func ResetCredential(rCtx RequestContext, user *models.Identity, newPassword str
 		return "", fmt.Errorf("get credential: %w", err)
 	}
 
+	hash, err := bcrypt.GenerateFromPassword([]byte(newPassword), bcrypt.DefaultCost)
```

Comment:
Why doesn't this hash come from the `GenerateFromPassword` function used elsewhere?

Reply:
The password in this case is generated so it doesn't need to be validated against the password policy